src/App.js

```diff
@@ -7,6 +7,7 @@ import {
   Route,
 } from 'react-router-dom';
 import apolloClient from './apollo';
+import NavBar from './Navbar';
```

Comment:
For consistency let's use the same name for the file and its default export, preferably:
`import Navbar from './Navbar';`